packages/integrations/vercel/README.md

```diff
@@ -101,7 +102,50 @@ import vercel from '@astrojs/vercel/serverless';
 export default defineConfig({
   output: 'server',
   adapter: vercel({
-    analytics: true,
+    webAnalytics: {
+      enabled: true,
+    },
+  }),
+});
+```
+
+#### `beforeSend`
+
+To define the `beforeSend` function, you need to create a separate file inside your root called `vercel-web-analytics.ts`.
+If you're not using TypeScript, you can define the function inside `vercel-web-analytics.js`.
```

Comment:
```suggestion
`beforeSend` is a function that can modify analytics event data before it's sent to Vercel.

Define this function in a separate file at the root of your project called `vercel-web-analytics.ts` (or `vercel-web-analytics.js` if you're not using TypeScript).
```

You've said in the section above not to pass `beforeSend` as a config option. Does this mean that simply having this file at the root of your project is enough? The Vercel adapter will automatically find it and use it? Is there anything else that needs to happen other than creating this file?

Reply:
Exactly, the user just needs to create the file and export the `beforeSend` function there